lagom-cloud-object-storage-example/account-impl/src/main/java/com/lightbend/lagom/account/impl/readside/AccountReportRepository.java

```diff
@@ -0,0 +1,79 @@
+package com.lightbend.lagom.account.impl.readside;
+
+import akka.Done;
+import com.google.common.base.Preconditions;
+import com.lightbend.lagom.account.impl.AccountEvent;
+import com.lightbend.lagom.account.impl.AccountEvent.DepositExecuted;
+import com.lightbend.lagom.account.impl.AccountEvent.WithdrawExecuted;
+import com.lightbend.lagom.javadsl.persistence.Offset;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public class AccountReportRepository {
+
+  private final Map<String, Report> reports = new HashMap<>();
```

Comment:
I this we must add a comment here that this is an in-memory hack and not use that in either a clustered environment or production.

Reply:
yes, that will be part of the README file

Reply:
I don't expect someone to use such an demo app in a production environment. 

With respect to clustered environment, the in-memory hack is not an issue in itself, but the fact that the tags are not sharded. As said above, I don't think we should go for a full blow Lagom demo with sharding / cluster, but demo the integration with Cloud Object Storage. But yes, we can review this strategy and make this demo app a little bit more robust.